flybywiresim/a32nx
# fix: Strobe lights tooltip

Fixes #5258

## Summary of Changes

This PR fixes the strobe light tooltip to give the correct positions.
## Screenshots (if necessary)


![Capture](https://user-images.githubusercontent.com/72807945/124343642-1265b000-db9b-11eb-91bc-18fc06126ba3.PNG)
## References





## Additional context



Discord username (if different from GitHub): Aviation Spotter#9102

## Testing instructions


1. Spawn (anywhere of choice)
2. Verify strobe light tooltips are correct
3. Verify other language settings do not have those tooltips broken (although they aren't translated)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
